src/js/pages/services/DeploymentsTab.js

```diff
@@ -124,6 +141,30 @@ class DeploymentsTab extends mixin(StoreMixin) {
     );
   }
 
+  renderAction(prop, deployment) {
```

Comment:
As mentioned in #411 this should be right before `renderEdit` 🔤  feel free to change it in either PR